src/components/SurveyList/SurveyItem.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @next/next/no-img-element */
+import Link from 'next/link';
+import Image from 'next/image';
+
+import { Survey } from 'services/survey';
+
+import styles from './SurveyList.module.css';
+
+interface SurveyItemProps {
+  survey: Survey;
+}
+
+const SurveyItem = ({ survey }: SurveyItemProps) => {
+  return (
+    <>
+      <img
+        src={survey.coverImageUrl}
+        className="w-full h-auto max-h-[360px] mb-8 rounded-lg"
+        alt={`${survey.id} survey image`}
```

Comment:
I think we can use the title here 
```suggestion
        alt={survey.title}
```

Reply:
Updated in [34a5526](https://github.com/hanam1ni/react-surveyor/pull/75/commits/34a5526aa6061380d83efb2e799b89da5a745cee)